description: somehow webview is missing files from chrome/browser in v88
author: Wengling Chen <feiyu2817@gmail.com>

---
 android_webview/lib/BUILD.gn |    2 ++
 1 file changed, 2 insertions(+)
--- a/android_webview/lib/BUILD.gn
+++ b/android_webview/lib/BUILD.gn
@@ -21,6 +21,8 @@ source_set("lib") {
     "//android_webview/gpu",
     "//android_webview/renderer",
     "//base",
+    "//chrome/browser:browser",
+    "//chrome/utility",
     "//cc/base",
     "//components/autofill/core/common",
     "//components/crash/core/common",
